app/controllers/application_controller.rb

```diff
@@ -2005,7 +2039,7 @@ def build_created_audit(rec, eh)
      :target_class => rec.class.base_class.name,
      :userid       => session[:userid],
      :message      => build_audit_msg(eh[:new], nil,
-                                      "[#{eh[:new][:name]}] Record created")
+                                      _("[%{name}] Record created") % {:name => eh[:new][:name]})
```

Comment:
Whatever goes into `audit.log` should stay in english, i.e. the gettext here needs to go away. There are other occurences like this in this PR.